js/app/historyStore.js

```diff
@@ -91,9 +92,11 @@ const HistoryStore = new Lang.Class({
         Dispatcher.get_default().register((payload) => {
             switch(payload.action_type) {
                 case Actions.HISTORY_BACK_CLICKED:
+                    this.get_previous_item().entry_point = EntryPoints.NAV_BUTTON_CLICKED;
```

Comment:
Are you suggesting that we make duplicate calls to the metrics daemon even though that causes warnings to print? Even if we don't care about that it makes the code messier because the code in `on_changed` dealing with the metrics can't easily be broken out into a function, because you need to use a different entry point just when the history buttons are clicked.